src/index.html

```diff
@@ -6,9 +6,54 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header class="container">
+      <a href="#" class="img__logo">
+        <img src="/src/images/logo.png"
+        class="logo" alt="logo">
```

Comment:
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/f7f91749-f377-411a-a08a-2b890d427204">
fix it everywhere